components/ThingsToDo.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import styled from "styled-components/native";
+import { Text, View, Vibration, TouchableOpacity } from 'react-native';
```

Comment:
I see that sometimes you use single quotes and sometimes double quotes - let's try to be consistent about it